linera-views/src/backends/scylla_db.rs

```diff
@@ -98,13 +112,38 @@ const MAX_BATCH_SIZE: usize = 5000;
 /// The keyspace to use for the ScyllaDB database.
 const KEYSPACE: &str = "kv";
 
+/// The default size of the cache for the load balancing policies.
+const DEFAULT_LOAD_BALANCING_POLICY_CACHE_SIZE: NonZeroUsize = NonZeroUsize::new(50_000).unwrap();
+
+enum LoadBalancingPolicyCacheEntry {
+    Ready(Arc<dyn LoadBalancingPolicy>),
+    // The timestamp of the last time the policy creation was attempted.
+    NotReady(Instant, Option<Token>),
+}
+
+/// The configuration of the ScyllaDB client.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct ScyllaDbClientConfig {
+    /// The delay before the sticky load balancing policy creation is retried.
+    pub delay_before_sticky_load_balancing_policy_retry_ms: u64,
+}
```

Comment:
Now that we have a `ScyllaDbClientConfig`, why not put the `DEFAULT_LOAD_BALANCING_POLICY_CACHE_SIZE` in it?